napari/_vispy/layers/vectors.py

```diff
@@ -12,33 +17,25 @@ def __init__(self, layer):
         self.layer.events.edge_color.connect(self._on_data_change)
 
         self.reset()
-        self._on_data_change()
+        # self._on_data_change()
```

Comment:
This needs to be addressed, along with line 17.

Reply:
I've uncommented the `on_data_change` to allow some methods to function (edge_color changes). There is still an open question of how we will handle these events that don't require an entire reslice event. I think there may be a reasonable path forward with the `SliceInput/Output` state implementation.